cpp/src/parquet/properties.h

```diff
@@ -264,8 +264,8 @@ class PARQUET_EXPORT WriterProperties {
       return this;
     }
 
-    /// Specify the max row group length.
-    /// Default 64M.
+    /// Specify the max number of rows to put in a single row group.
+    /// Default 64Mi rows.
```

Comment:
```suggestion
    /// Default 1Mi rows.
```

Reply:
Ah, good catch.  Thanks.  I've made this change.